app.js

```diff
@@ -95,9 +129,9 @@ const config = {
 if (!process.env.AWS_LAMBDA_FUNCTION_NAME) {
   const port = process.env.PORT || 3001;
 
-  app.listen(port, function () {
-    console.info('Server started on port ' + port)
-  })
+  app.listen(port, () => {
+    console.info(`Server started on port ${port}`); // eslint-disable-line no-console
```

Comment:
here and above you're using backtics.  I remember production having trouble with backtics on another pice of code, because of js version running?

Reply:
Good thinking, but we're in the clear with this microservice.  The back-tick error was in the Rails app.  This app has already had functioning back-ticks.